ray-operator/test/sampleyaml/rayservice_test.go

```diff
@@ -43,18 +43,19 @@ func TestRayService(t *testing.T) {
 			g.Expect(err).NotTo(HaveOccurred())
 			g.Expect(rayService).NotTo(BeNil())
 
-			var rayClusterName string
+			test.T().Logf("Waiting for RayService %s/%s to running", rayService.Namespace, rayService.Name)
+			g.Eventually(RayService(test, rayService.Namespace, rayService.Name), TestTimeoutMedium).
+				Should(WithTransform(RayServiceStatus, Equal(rayv1.Running)))
+			// Get the latest rayService
+			rayService, err = GetRayService(test, namespace.Name, rayServiceFromYaml.Name)
+			g.Expect(err).NotTo(HaveOccurred())
+			g.Expect(rayService).NotTo(BeNil())
+
 			// Wait for RayCluster name to be populated
-			g.Eventually(func(g Gomega) {
-				rs, err := GetRayService(test, namespace.Name, rayServiceFromYaml.Name)
-				g.Expect(err).NotTo(HaveOccurred())
-				if rs.Status.PendingServiceStatus.RayClusterName != "" {
-					rayClusterName = rs.Status.PendingServiceStatus.RayClusterName
-				} else if rs.Status.ActiveServiceStatus.RayClusterName != "" {
-					rayClusterName = rs.Status.ActiveServiceStatus.RayClusterName
-				}
-				g.Expect(rayClusterName).NotTo(BeEmpty())
-			}, TestTimeoutShort).Should(Succeed())
+			g.Eventually(RayService(test, namespace.Name, rayServiceFromYaml.Name), TestTimeoutShort).Should(
+				WithTransform(UnderlyingRayClusterName, Not(BeEmpty())),
```

Comment:
We can get `ActiveServiceStatus.RayClusterName` directly because the RayService has become `Running`, and zero-downtime has not been triggered.

Reply:
Ok, I will also remove UnderlyingRayClusterName, because this is not used anymore